src/olympia/files/tests/test_utils_.py

```diff
@@ -490,3 +490,13 @@ def test_bump_version_in_package_json(file_obj):
         utils.update_version_number(file_obj, '0.0.1.1-signed')
         parsed = utils.parse_xpi(file_obj.file_path)
         assert parsed['version'] == '0.0.1.1-signed'
+
+
+def test_bump_version_in_manifest_json(file_obj):
```

Comment:
Indeed. It has its separate commit though. I added it to this PR because that's when I discovered the missing piece of code (that we might never need though...)